[0001].[2021.04.03.00.45.25.799526].SRV1.INF:Main.py.<module>.298:Main Program Started
[0002].[2021.04.03.00.45.25.800527].SRV1.ERR:Main.py.<module>.301:Artificial Error Message for testing purpose
[0003].[2021.04.03.00.45.25.801028].SRV1.INF:Main.py.<module>.302:Command Line iniFileName=default.ini
[0004].[2021.04.03.00.45.25.802028].SRV1.INF:Main.py.<module>.304:Command Line VerboseMode=None
[0005].[2021.04.03.00.45.25.802529].SRV1.INF:Main.py.<module>.307:AppSpoManager.DebugVerboseModeOn=True
[0006].[2021.04.03.00.45.25.803529].SRV1.INF:Main.py.<module>.308:AppSpoManager.DebugInfoMessages=True
[0007].[2021.04.03.00.45.25.804030].SRV1.INF:Main.py.<module>.309:AppSpoManager.DebugErrorMessages=True
[0008].[2021.04.03.00.45.25.804530].SRV1.INF:Main.py.<module>.310:AppSpoManager.DebugTargetConsole=True
[0009].[2021.04.03.00.45.25.805530].SRV1.INF:Main.py.<module>.311:AppSpoManager.DebugTargetFile=True
[0010].[2021.04.03.00.45.25.806031].SRV1.INF:Main.py.<module>.312:AppSpoManager.DebugTargetFilePath=Debugs\Debug1
[0011].[2021.04.03.00.45.25.807032].SRV1.INF:Main.py.<module>.313:AppSpoManager.DebugLinesPerFile=1000
[0012].[2021.04.03.00.45.25.807532].SRV1.INF:Main.py.<module>.314:AppSpoManager.AppRunFolder=c:\Apps\PythonTry1
[0013].[2021.04.03.00.45.25.808032].SRV1.INF:Main.py.<module>.337:ModuleFileNameOnly=Main.py
[0014].[2021.04.03.00.45.25.809033].SRV1.INF:Main.py.<module>.345:File=c:\Apps\PythonTry1\Main.py
[0015].[2021.04.03.00.45.25.809533].SRV1.INF:Main.py.<module>.346:FileDirName=c:\Apps\PythonTry1
[0016].[2021.04.03.00.45.25.810033].SRV1.INF:Main.py.<module>.347:FileAbsPath=c:\Apps\PythonTry1\Main.py
[0017].[2021.04.03.00.45.25.811034].SRV1.INF:Main.py.<module>.350:ModuleName1=__main__
[0018].[2021.04.03.00.45.25.811534].SRV1.INF:Main.py.<module>.351:ModuleName2=Main
[0019].[2021.04.03.00.45.25.812535].SRV1.INF:Main.py.<module>.360:Main Program Started
[0020].[2021.04.03.00.45.25.813035].SRV1.INF:Main.py.<module>.361:Python Executable Path=C:\Program Files\Python39\python.exe
[0021].[2021.04.03.00.45.25.813536].SRV1.INF:Main.py.<module>.362:Python Interpreter Version=3.9.2 (tags/v3.9.2:1a79785, Feb 19 2021, 13:44:55) [MSC v.1928 64 bit (AMD64)]
[0022].[2021.04.03.00.45.25.814536].SRV1.INF:Main.py.<module>.365:Sys.Path=['c:\\Apps\\PythonTry1', 'C:\\Program Files\\Python39\\python39.zip', 'C:\\Program Files\\Python39\\DLLs', 'C:\\Program Files\\Python39\\lib', 'C:\\Program Files\\Python39', 'C:\\Users\\user2\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\Program Files\\Python39\\lib\\site-packages']
[0023].[2021.04.03.00.45.25.844058].SRV1.INF:Main.py.<module>.412:Showing MainWindow
